### PR TITLE
Add clickable diffs to recent changes view

### PR DESCRIPTION
**Pull request by danilax86**
image

### CODE REVIEW

File: history/view.qtpl [hunk 1]
```diff
@@ -13,6 +13,24 @@ HyphaeLinksHTML returns a comma-separated list of hyphae that were affected by t
...
+    {% code entries := rev.hyphaeAffected() %}
+{% stripspace %}
+    {% if len(entries) == 1 %}
+        <a href="/primitive-diff/{%s rev.Hash %}/{%s entries[0] %}">{%s rev.Hash %}</a>
+    {% else %}
+        {% for i, hyphaName := range entries %}
+            {% if i > 0 %}
+                <span aria-hidden="true">, </span>
+            {% endif %}
+                <a href="/primitive-diff/{%s rev.Hash %}/{%s hyphaName %}">{%s rev.Hash %}({%s hyphaName %})</a>
```
Comment (bouncepaw) [inline history/view.qtpl:28]: I think you should have a space before the opening parenthesis.

hashhash (hypha)

Comment (danilax86) [reply history/view.qtpl]: Inserted non-breaking one

File: history/view.qtpl [hunk 2]
```diff
@@ -13,6 +13,24 @@ HyphaeLinksHTML returns a comma-separated list of hyphae that were affected by t
 {% endstripspace %}
 {% endfunc %}
 
+
+HyphaeDiffsHTML returns a comma-separated list of diffs links of current revision for every affected file as HTML string.
+{% func (rev Revision) HyphaeDiffsHTML() %}
```
Comment (bouncepaw) [inline history/view.qtpl:18]: Снимок экрана 2024-03-10 в 22 23 57

Seen this in action. Works as expected, thank you!

However, I don't like this duplication of the hash. Maybe show it one time only? Something like that maybe:

Снимок экрана 2024-03-10 в 22 25 33

**Review by bouncepaw — CHANGES_REQUESTED** [review]

File: history/view.qtpl (hunk 2 above)
Comment (danilax86) [reply history/view.qtpl:18]: This variant looks cleaner. Done!

image

**Pull request merged**